languages/go/oso.go

```diff
@@ -150,3 +150,7 @@ func (o Oso) IsAllowed(actor interface{}, action interface{}, resource interface
 		return false, nil
 	}
 }
+
+func (o Oso) Repl() error {
```

Comment:
Can we add a comment above for API docs?

Reply:
Yeah